src/components/Table.js

```diff
@@ -172,13 +183,13 @@ export class Table extends Component {
             showSearch,
             showPagination,
             CustomPagination,
+            isServerPagination,
```

Comment:
This will be null as you've added the `isServerPagination` to the `pagination` object